vendor.conf

```diff
@@ -13,10 +14,11 @@ github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76 # v1.1.0
 github.com/dgrijalva/jwt-go a2c85815a77d0f951e33ba4db5ae93629a1530af
 github.com/docker/distribution 83389a148052d74ac602f5f1d62f86ff2f3c4aa5
 github.com/docker/docker 6e3113f700dea1bf2785d94731b4b5a1e602d9ab
+github.com/docker/compose-on-kubernetes 939a5ba6c6dbfc78e6cc29b612716d5688b9a323
 github.com/docker/docker-credential-helpers 5241b46610f2491efdf9d1c85f1ddf5b02f6d962
 # the docker/go package contains a customized version of canonical/json
 # and is used by Notary. The package is periodically rebased on current Go versions.
-github.com/docker/go d30aec9fd63c35133f8f79c3412ad91a3b08be06
+github.com/docker/go 0ebda48a7f143b1cce9eb37a8c1106ac762a3430 # v0.34.0
```

Comment:
Looks like you updated the wrong line here;  this tag/commit is for `cloud.google.com/go`

Probably that caused the huge diff in this commit

<img width="251" alt="screen shot 2018-12-20 at 14 30 17" src="https://user-images.githubusercontent.com/1804568/50287939-c29fbd00-0464-11e9-81f2-e75e96b03950.png">